core/internal-groups.rst

```diff
@@ -0,0 +1,44 @@
+Internal Groups and Areas of Focus
+---------------------------------
+
+Core is probably the big component in coala. Almost everything else is heavily
+tied with it; so it doesn't make sense to have a huge, monolithic, and
+unfocused team. So I propose to have a few areas of focus that each team member
+can choose to align themselves with.
+
+Please note that these are not strict teams. And by no means exhaustive. These
+is simply a categorization to co-ordinate efforts better. And it is entirely
+possible than an issue doesn't fall under one of the proposed groups squarely -
+overlaps are allowed. In short, these is simply a way to organize better, in an
+undivisive manner.
```

Comment:
not sure this makes sense, if something's too big, we may want another team. Also for an internal group you need some kind of "leader" to coordinate efforts, you have multiple people in the subgroup, so we have the same result as having another normal team. So better split up into multiple teams like `Core-Scheduler`, `Core-Bears` etc, or if you just want to have more manpower while keeping flexibility, name those groups `Core-1`, `Core-2`.